src/test/resources/application.properties

```diff
@@ -5,4 +5,6 @@ api.url=http://api.chs.local:4001
 account.api.url=http://account.chs.local:4001
 chs.internal.api.key=dummy
 
-de.flapdoodle.mongodb.embedded.version=6.0.16-rc0
\ No newline at end of file
+de.flapdoodle.mongodb.embedded.version=6.0.16-rc0
+
+signin.url=http://chs.local/signin
```

Comment:
Will this be overridden on the relevant environments? Just might need to make sure that's done/added appropriately

Reply:
this is test properties, main properties will be overriden using Vault values/

Reply:
The application.properties file in src/main/resources uses this variable `signin.url=${CHS_URL}/signin`, so it will be overriden on the relevant environments.

The file we're commenting on is from the test variant for application.properties i.e. in src/test/resources, so is only used for the automated tests.